### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by MichaelPFrey**

### CODE REVIEW

File: releases/2018.XX.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Added $preview variable which is set to true in preview mode (#1963)
 * Removed glide() and subdiv() which were never implemented
 * Added is_undef() to look up variables silently (#2535)
+* Added is_list(), is_num, is_bool, is_string
```
Comment (kintel) [inline releases/2018.XX.md]: Probably add `()` behind all functions for consistency

Comment (MichaelPFrey) [reply releases/2018.XX.md]: thank you for the input - updated

**Comment by MichaelAtOz** [comment]
Now that the download page shows 2019.01-RC2 should it be announced on the forum so more people take it for a spin?

**Comment by t-paul** [comment]
I'd say yes. I meant to put out a post on the website, but didn't find the time for that yet. So quick forum post might be a good thing.

**Comment by MichaelAtOz** [comment]
Forum draft (with links in the real thing)

> New Release of OpenSCAD, soon - please test RC
----------------------------------------------
>Well it is with great anticipation, if you have been following GitHub, that a new Release would be forthcoming, since the last magnificent 2015 Release.
>
>You may have noticed on the Downloads page, a little thing OpenSCAD-2019.01-RC2, available for Windows & Linux. Mac lagging as usual ;)
>
>You can help to ensure a better final release by giving the Release Candidate a spin with your more complex models.
>Please report any issue here, or on GitHub if you're sure it's a bug and you have access.
>
>The changes are listed here.
>
>Note that there is now significant syntax checking with Warning Messages, so if you have been using some dodgy methods don't get a shock the first time you compile. Even the best of us has discovered a few termites lurking. 

There 'here' is https://github.com/openscad/openscad/blob/master/releases/2018.XX.md
do we want to rename that 2019.XX before announcing?

**Comment by t-paul** [comment]
Oh, oops, this was not merged yet. I somewhat optimistically renamed the file to https://github.com/openscad/openscad/blob/master/releases/2019.02.md (well, that's the target if we want  to get into the next Debian/Ubuntu releases).

**Comment by t-paul** [comment]
Ok, I've imported the changes manually as the file rename seems to have broken the automatic github merge.

Forum note looks good to me - except the *Mac lagging as usual* part which I believe is not true, I think @kintel just did not name it as RC2 but it should be either exactly the RC2 tag, or at least the same day?

**Comment by kintel** [comment]
The latest Mac binaries was built from the RC2 branch, but I didn't edit the version number as we don't have a script for that and manual tinkering with binaries scare me :)

**Comment by t-paul** [comment]
That's what I remembered, so maybe we can change the display on the download page, but otherwise all "snapshot" downloads are in fact RC2.

**Comment by t-paul** [comment]
Also if all goes as planned, RC2 should migrate to Debian/Testing in a couple of hours... :crossed_fingers: 

=> [Package Tracker](https://tracker.debian.org/pkg/openscad)

**Comment by MichaelAtOz** [comment]
> I somewhat optimistically renamed the file to https://github.com/openscad/openscad/blob/master/releases/2019.02.md (well, that's the target if we want to get into the next Debian/Ubuntu releases).

I held off the forum announcement due to the blanking issue (#2781) which looked bad, but as no one else seems to have that problem, I'll announce it.
But as there are 7 days in 02 is it going to go as 2019.02 even if it slips into .03?
And/or should there be a RC3?

**Comment by MichaelAtOz** [comment]
> 
> 
> > I somewhat optimistically renamed the file to https://github.com/openscad/openscad/blob/master/releases/2019.02.md (well, that's the target if we want to get into the next Debian/Ubuntu releases).
> 
> I held off the forum announcement due to the blanking issue (#2781) which looked bad, but as no one else seems to have that problem, I'll announce it.
> But as there are 7 days in 02 is it going to go as 2019.02 even if it slips into .03?
> And/or should there be a RC3?

@t-paul @kintel ??  ...4 days

**Comment by kintel** [comment]
I don't really think the naming matters all that much, so I'd be ok with keeping naming at 2019.02, even if the release technically is in March.

**Pull request closed without merging**